### ISSUE TITLE
Check last years questions marks in Annex Table

### ISSUE THREAD

**Issue by odontaster** [slarge/fisheryO#55]
For multi annual advice, the F and SSB symbols should be also in the last year, instead of the question mark, right?

**Comment by sarahlmillar** [comment]
Yes, they should be carries over from the previous year into the last year .

**Comment by odontaster** [comment]
ok, done, just have a doubt with the spurdog, not sure if the symbols should be carried from 2016. Please check and let me know.

**Comment by sarahlmillar** [comment]
No spurdog has an assessment this year so it should have new symbols. The SAG have not been updated for EF yet. Need to speak with Inigo to push this along
